.github/workflows/ci.yml

```diff
@@ -294,17 +300,19 @@ jobs:
       - name: Push to us.gcr.io
         run: |
           set -euxo pipefail
-          docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/relay:$REVISION"
-          docker push "us.gcr.io/sentryio/relay:$REVISION"
+          docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/${{ matrix.image_name }}:$REVISION"
+          docker push "us.gcr.io/sentryio/${{ matrix.image_name }}:$REVISION"
 
       - name: Push nightly to us.gcr.io
         if: github.ref == 'refs/heads/master'
         run: |
           set -euxo pipefail
-          docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/relay:nightly"
-          docker push "us.gcr.io/sentryio/relay:nightly"
+          docker tag "$IMG_VERSIONED" "us.gcr.io/sentryio/${{ matrix.image_name }}:nightly"
+          docker push "us.gcr.io/sentryio/${{ matrix.image_name }}:nightly"
 
       - name: Upload gocd deployment assets
+        # Collect assets only for processing Relay, since it contains more information.
+        if: matrix.image_name == 'relay'
```

Comment:
In the future, not only images will be different but also assets, including debug symbols. Should we upload both, even if relay-pop is a duplicate for now?

Reply:
I would say it doesn't matter now and we can revisit this later. 
Since it will also require more changes, which I would better address in the followups.